requirements_all.txt

```diff
@@ -2850,7 +2850,7 @@ wirelesstagpy==0.8.1
 wled==0.17.0
 
 # homeassistant.components.wolflink
-wolf-smartset==0.1.11
+wolf-comm==0.0.4
```

Comment:
There's still no LICENSE file in the GitHub repo.

The tag number version also doesn't match the pypi version. Please make sure that they match. I recommend following semantic version specifier.

https://github.com/janrothkegel/wolf-comm/releases/tag/v0.0.04

https://pypi.org/project/wolf-comm/0.0.4/

Also, the version in setup.py hasn't been updated:
https://github.com/janrothkegel/wolf-comm/blob/77ac8ea839d376890f752023d9b22eed7d007915/setup.py#L8

Reply:
Sorry, I missed to synchronize the latest commits with GitHub. The LICENSE.txt was only in PyPI but now is also in the repository.